### PR TITLE
Added class creation

### PR DESCRIPTION
**Pull request by joelwdav**
doesn't include the stuff from the sign in branch.

path '/classes' is the page to add and remove the classes.

### CODE REVIEW

File: config/routes.rb~ [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
```
Comment (anton5798) [inline config/routes.rb~]: can remove this ~ file

**Review by anton5798 — CHANGES_REQUESTED** [review]
Generally looks good! A couple of comments 

1. Remove ~ file
2. Could you please rename the controller/view to be courses_controller to be consistent with the model name?
3. Also I think the link that appears after the creation of class is incorrect 
4. We might also want the show method in the controller so that the new link actually works
5. Remove the testing db files. They should be .gitignored, and I'm not sure why they are being added.

**Comment by joelwdav** [comment]
I changed the routes and controllers but I didn't do anything with the links because I don't know what I'm doing with them yet. I put them there just to make sure things were working properly.

**Review by anton5798 — APPROVED** [review]
Looks good! Still minor details to be fixed, but we'll do that once we have more functionality

**Pull request merged**